packages/material-ui-lab/src/DayPicker/DayPicker.tsx

```diff
@@ -193,7 +193,6 @@ const DayPicker = React.forwardRef(function DayPicker<
       <FadeTransitionGroup
         reduceAnimations={reduceAnimations}
         className={classes.viewTransitionContainer}
-        transKey={openView}
```

Comment:
Does it introduce a regression on the problem the prop was fixing? If not why?

Reply:
Let me check another course of action, regarding logic. I would check it 😃  Thanks!

Reply:
I have noticed this issue as well in [Landscape Orientation](https://next.material-ui.com/components/date-picker/#landscape-orientation) and others, for example in [Custom input component](https://next.material-ui.com/components/date-picker/#custom-input-component) jumps to the begging of the page. As well in [Time Picker](https://next.material-ui.com/components/time-picker/#basic-usage). I think the issue is the logic used in [useViews.tsx](https://github.com/mui-org/material-ui/blob/next/packages/material-ui-lab/src/internal/pickers/hooks/useViews.tsx). But I am not sure, let me keep looking for an optimal solution. 😺